### PR TITLE
Update docs to specify that we support k8s version 1.16 and higher

### PR DESCRIPTION
**Pull request by viveksinghggits**
## Change Overview

Since we recently upgraded the CRD APIs to `v1` from `v1beta1` and `v1` APIs are not available in Kubernetes versions lower than 1.16; we will have to update our docs as well because they still said we support k8s version 1.8.

## Pull request type

Please check the type of change your PR introduces:

- [x] :world_map: Documentation

## Issues

- #XXX

## Test Plan

```
make docs
```

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Installing Kanister
 Prerequisites
 =============
 
-* Kubernetes 1.8 or higher with Beta APIs enabled
+* Kubernetes 1.16 or higher
```
Comment (pavannd1) [inline docs/install.rst:13]: Should we add a note saying they can install 0.60.0 in case they want < 1.15 ?

Comment (viveksinghggits) [reply docs/install.rst:13]: right, we should do that. I will make the changes.

File: docs/install.rst [hunk 2]
```diff
@@ -10,7 +10,8 @@ Installing Kanister
 Prerequisites
 =============
 
-* Kubernetes 1.8 or higher with Beta APIs enabled
+* Kubernetes ``1.16`` or higher. For cluster version lower than ``1.16``,
+  we recommend installing Kanister version lower than ``0.63.0``
```
Comment (pavannd1) [inline docs/install.rst:14]:
```suggestion
  we recommend installing Kanister version ``0.62.0`` or lower.
```

File: docs/tutorial.rst [hunk 3]
```diff
@@ -13,7 +13,8 @@ of Kanister's features to manage the application's data.
 Prerequisites
 =============
 
-* Kubernetes 1.8 or higher with Beta APIs enabled
+* Kubernetes ``1.16`` or higher. For cluster version lower than ``1.16``,
+  we recommend installing Kanister version lower than ``0.63.0``
```
Comment (pavannd1) [inline docs/tutorial.rst:17]:
```suggestion
  we recommend installing Kanister version ``0.62.0`` or lower.
```

**Review by pavannd1 — APPROVED** [review]

**Pull request merged**